package.json

```diff
@@ -6,22 +6,23 @@
     "grunt": "^1.0.1",
     "grunt-cli": "1.2.0",
     "grunt-contrib-concat": "1.0.1",
-    "grunt-contrib-jshint": "1.0.0",
+    "grunt-contrib-jshint": "1.1.0",
     "grunt-contrib-watch": "1.0.0",
-    "grunt-exec": "1.0.0",
+    "grunt-exec": "3.0.0",
     "grunt-karma": "2.0.0",
-    "jasmine": "^2.0.1",
-    "jasmine-core": "^2.4.1",
-    "jasmine-reporters": "2.1.1",
-    "karma": "^1.2.0",
+    "jasmine": "^2.8.0",
+    "jasmine-core": "^2.8.0",
+    "jasmine-reporters": "2.3.0",
+    "karma": "^2.0.0",
+    "karma-chrome-launcher": "^2.2.0",
     "karma-coverage": "^1.1.1",
-    "karma-jasmine": "^1.0.2",
-    "karma-junit-reporter": "0.2.2",
+    "karma-jasmine": "^1.1.1",
+    "karma-junit-reporter": "1.2.0",
     "karma-ng-html2js-preprocessor": "^1.0.0",
-    "karma-phantomjs-launcher": "^1.0.1",
-    "karma-teamcity-reporter": "^1.0.0",
-    "phantomjs-prebuilt": "^2.1.12",
-    "rimraf": "2.5.4"
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-teamcity-reporter": "^1.1.0",
+    "phantomjs-prebuilt": "^2.1.16",
```

Comment:
Do you still need Phantom if you are switching to ChromeHeadless?

Reply:
Good spot. Will remove.